### PR TITLE
Add attachable option in create_network

### LINKED ISSUE #1336: Support Attachable flag to network create

### PR DESCRIPTION
**Pull request by shin-**
Picking up @dnephin 's work in #1337 

Resolves #1336

### CODE REVIEW

**Comment by shin-** [comment]
@dnephin If you have a minute can you do a quick review of the changes here?

File: tests/helpers.py [hunk 1]
```diff
@@ -73,4 +73,4 @@ def force_leave_swarm(client):
             if e.explanation == "context deadline exceeded":
                 continue
             else:
-                raise
+                return
```
Comment (dnephin) [inline tests/helpers.py]: What's this change about?

File: tests/integration/models_swarm_test.py [hunk 2]
```diff
@@ -19,4 +19,7 @@ def test_init_update_leave(self):
         assert client.swarm.leave(force=True)
         with self.assertRaises(docker.errors.APIError) as cm:
             client.swarm.reload()
-        assert cm.exception.response.status_code == 406
+        assert (
+            cm.exception.response.status_code == 406 or
+            cm.exception.response.status_code == 503
```
Comment (dnephin) [inline tests/integration/models_swarm_test.py]: Comment why it would be one or the other?

File: tests/unit/api_network_test.py [hunk 3]
```diff
@@ -104,6 +104,10 @@ def test_create_network(self):
                     }
                 })
 
+    @requires_api_version('1.24')
+    def test_create_network_with_attachable(self):
+        pass
```
Comment (dnephin) [inline tests/unit/api_network_test.py]: Sorry I added this empty test stub. I think we can remove it now that there are other tests.

File: tests/helpers.py (hunk 1 above)
Comment (shin-) [inline tests/helpers.py]: It was overzealous - we don't really care if there's an error during the `tearDown`

**Pull request merged**